WcaOnRails/lib/dues_calculator.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module DuesCalculator
+  def self.update_exchange_rates_if_needed
+    if !Money.default_bank.rates_updated_at || Money.default_bank.rates_updated_at < 1.day.ago
+      Money.default_bank.update_rates
+    end
+  end
+
+  def self.dues_per_competitor_in_usd(country_iso2, base_entry_fee_lowest_denomination, currency_code)
```

Comment:
You are passing in the `currency_code` in order to be able to convert to USD internally.
Would it make sense to convert back to the local `currency_code` currency before returning?

Reply:
In other words, are there use cases in the code where it makes sense to have the Dues always be provided in USD?

Reply:
Yes there is, For WFC export, we want the dues in USD.

Reply:
Okay, then just return them flat in USD and let other consumers of this method handle conversions the way they want.